addons/account_check_printing/tests/test_print_check.py

```diff
@@ -60,21 +60,26 @@ def test_print_check(self):
         invoices = self.env['account.move']
         for i in range(0, 15):
             invoices |= self.create_invoice(is_refund=(i % 3 == 0))
-        payment = self.create_payment(invoices)
```

Comment:
do not modify tests if not necessary

Reply:
We need to adapt the test, as it is creating a single payment for both invoices and credit notes (and thus several receivable/payable destination accounts)
Before this commit it was silently wrong, but will crash now due to the new UserError message